spec/models/checkout_spec.rb

```diff
@@ -133,4 +125,75 @@
       expect(checkout).to be_shadowed
     end
   end
+
+  describe '#status_human' do
+    context 'when an item is claims returned' do
+      it 'returns the right status_human' do
+        checkout.record['fields']['overdue'] = true
+        checkout.record['fields']['claimsReturnedDate'] = '2019-12-01'
+        expect(checkout.status_human).to eq 'Claims Returned'
+      end
+    end
+
+    context 'when an item is overdue' do
+      it 'returns the right status_human' do
+        checkout.record['fields']['overdue'] = true
+        expect(checkout.status_human).to eq 'Overdue'
+      end
+    end
+
+    context 'when an item is not overdue' do
+      it 'returns an empty status_human' do
```

Comment:
```suggestion
      it 'is nil' do
```

thinking that "empty" can mean other things